packages/falso/src/lib/core/unique-validators.ts

```diff
@@ -0,0 +1,32 @@
+export const primitiveValueIsUnique: <T>(item: T, items: T[]) => boolean = (
+  item,
+  items
+) => !items.includes(item);
+
+export const dateIsUnique: (date: Date, dates: Date[]) => boolean = (
+  date,
+  dates
+) => !dates.some((d) => d.valueOf() === date.valueOf());
+
+export const objectWithIdIsUnique: <T extends { id: string }>(
+  item: T,
+  items: T[]
+) => boolean = (item, items) => objectIsUnique(item, items, ['id']);
+
+export const objectIsUnique: (
+  item: any,
```

Comment:
Why any and not a generic?

Reply:
Yeah I wasn't a fan of this  either. On further inspection i've managed to change it to `objectIsUnique<T extends Record<string, any>>`. I think this should be ok for now